integration-tests/mongodb-client/src/main/resources/application.properties

```diff
@@ -1,4 +1,7 @@
 quarkus.mongodb.connection-string=mongodb://localhost:27017
 quarkus.mongodb.write-concern.journal=false
 quarkus.mongodb.parameter-injection.connection-string=mongodb://localhost:27017
-quarkus.mongodb.parameter-injection.write-concern.journal=false
\ No newline at end of file
+quarkus.mongodb.parameter-injection.write-concern.journal=true
+
+quarkus.mongodb.dev-services.write-concern.journal=false
+quarkus.mongodb.dev-services2.write-concern.journal=false
```

Comment:
what do you think of this @geoand?  And the `BookResource` change above.  It's not the "cleanest" but I was trying to find a simple way to test both the `MongoTestResource` driven stuff and the dev services as well and this is the simplest I could make it.

Reply:
I like the idea.